webhooks-extension/docs/GettingStarted.md

```diff
@@ -0,0 +1,161 @@
+
+This field specifies the service account that will be used by the PipelineRun. It should be the same service account that you set up RBAC permissions for in the section 'Setting up a webhook from scratch' and that you patched credentials onto in the sections, 'Create credentials: Git' and 'Create credentials: Docker', above. 
+
+### Docker Registry
+
+Finally, select the Docker registry that any built images should be pushed to. This is the docker registry that the 'docker' credentials earlier are associated with. 
+
+## Putting it all together: test it's working
+
+Once a webhook is set up, a `git push` or merged Pull Request to a branch in the monitored repository should trigger the creation of a correct PipelineRun. This PipelineRun will show up in the Tekton dashboard as usual. As of August 2019, a successful webhook will trigger pod changes in the target namespace of the form shown below. 
+
+Firstly, webhook receipt will drive a pod whose name corresponds to the 'Name' field of the webhook. E.g. for a Name of `buildah-hook` we see, 
+
+```
+buildah-hook-xwxm5-rp6s6-deployment-6674f7fdbb-pdgmw   0/2   Pending   0     0s
+buildah-hook-xwxm5-rp6s6-deployment-6674f7fdbb-pdgmw   0/2   Pending   0     0s
+buildah-hook-xwxm5-rp6s6-deployment-6674f7fdbb-pdgmw   0/2   ContainerCreating   0     0s
+buildah-hook-xwxm5-rp6s6-deployment-6674f7fdbb-pdgmw   1/2   Running   0     4s
+buildah-hook-xwxm5-rp6s6-deployment-6674f7fdbb-pdgmw   2/2   Running   0     4s
+```
+
+Next this pod will trigger the webhooks extension 'sink': 
+
+```
+webhooks-extension-sink-dqxkm-deployment-5f64979c5b-8dk4k   0/2   Pending   0     0s
+webhooks-extension-sink-dqxkm-deployment-5f64979c5b-8dk4k   0/2   Pending   0     0s
+webhooks-extension-sink-dqxkm-deployment-5f64979c5b-8dk4k   0/2   ContainerCreating   0     0s
+webhooks-extension-sink-dqxkm-deployment-5f64979c5b-8dk4k   0/2   Running   0     6s
+webhooks-extension-sink-dqxkm-deployment-5f64979c5b-8dk4k   1/2   Running   0     6s
+webhooks-extension-sink-dqxkm-deployment-5f64979c5b-8dk4k   2/2   Running   0     6s
+```
+
+The 'sink' will then create the expected PipelineRun:
+
+```
+buildah-hook-1565009498-build-simple-6qc72-pod-1f0a5e   0/6   Pending   0     1s
+buildah-hook-1565009498-build-simple-6qc72-pod-1f0a5e   0/6   Pending   0     1s
+buildah-hook-1565009498-build-simple-6qc72-pod-1f0a5e   0/6   Init:0/3   0     1s
+buildah-hook-1565009498-build-simple-6qc72-pod-1f0a5e   0/6   Init:1/3   0     4s
+buildah-hook-1565009498-build-simple-6qc72-pod-1f0a5e   0/6   Init:2/3   0     9s
+buildah-hook-1565009498-build-simple-6qc72-pod-1f0a5e   0/6   PodInitializing   0     10s
+buildah-hook-1565009498-build-simple-6qc72-pod-1f0a5e   5/6   Running   0     37s
+buildah-hook-1565009498-build-simple-6qc72-pod-1f0a5e   3/6   Running   0     2m38s
+buildah-hook-1565009498-build-simple-6qc72-pod-1f0a5e   2/6   Running   0     2m56s
+buildah-hook-1565009498-build-simple-6qc72-pod-1f0a5e   0/6   Completed   0     2m57s
```

Comment:
In this example I see the number of containers running decreasing, from 5/6 to 3/6 to 2/6 and 0/6, is it worth a sentence explaining what this means? Is it simply running the tasks and as they're done, the number on the left decreases so there's nothing to do (hence `completed`)?